migration/init.go

```diff
@@ -58,5 +58,7 @@ func (Initializer) FromGenesis(opts weave.Options, params weave.GenesisParams, k
 		}
 	}
 
+	MustInitPkg(kv, "cron")
```

Comment:
I think this is bad approach. Initialization should be done in the genesis file.
This creates a dependency and hides a single package initialization in unrelated code.

Reply:
@husio we are fixing this one off, please read a Slack message. It's not for master branch of production use other than extracting data from the testnet.

Reply:
I am confused. I thought you want me to review it. I don't think this is not something that can get into `master`.